### PR TITLE
Update plotting episode to use the dataset

### LINKED ISSUE #72: plots are only on random data
Introducing random data generation to teach plotting instead of working with the data doesn't seem like the best strategy.  We found the following plots to be useful examples.

Numerical histogram:
df['C06_rooms'].plot(kind='hist')

Categorical histogram:
df.groupby('C02_respondent_wall_type')['C01_respondent_roof_type'].hist(stacked=False)

Scatter:
df.plot.scatter('gps:Latitude','gps:Longitude',c='gps:Altitude',colormap='jet')

2 exercises
df.plot.scatter('A11_years_farm','B16_years_liv',c='C05_buildings_in_compound',colormap='jet')
challenge
df.groupby('C02_respondent_wall_type')['C06_rooms'].mean().plot(kind='bar')

### PR DESCRIPTION
**Pull request by brownsarahm**
closes #72 

This also assumes a copy of the dataset with shorter variable names (the alphanumeric codes preceding the meaningful name removed).  I would advocate we promote use of a simpler naming convention throughout but I could alternatively change the variable names here.  Please advise.

### CODE REVIEW

**Comment by sechilds** [comment]
There is some odd formatting in this PR and I don't know about re-writing a section to talk about other plotting libraries.

**Comment by brownsarahm** [comment]
I will render locally and check formatting and I can take out the seaborn example if that's important, but I strongly think that the episode should be rewritten to use the SAFI dataset rather than introduce random number generation for plotting.

File: _episodes/13-matplotlib.md [hunk 1]
```diff
@@ -14,286 +14,180 @@ keypoints:
...
+~~~
+safi_df['years_liv'].hist(bins=20)
+~~~
+{: .language-python}
 
 
-~~~
-import numpy as np
-import pandas as pd
+We can also specify the column as a parameter and a groupby column with the `by` keyword. there are a lot of keywords available to make it look better, we can see some of the most likely ones (as decided by pandas developers) by using <kbd>shift</kbd> + <kbd>tab<kbd>. Lets try `layout`, `figsize`, and `sharex`.
```
[the remark this answers is not in the document]
Comment (katrintirok) [reply _episodes/13-matplotlib.md]: @sechilds I added the missing / in line 64, since I am also working with @brownsarahm 's repo

**Comment by katrintirok** [comment]
@sechilds and @brownsarahm I made some more updates on this PR for the plotting episode, please have a look and suggest any changes necessary. I added files to show created plots to the fig directory. I added solutions for exercises. I checked the format of exercises, key words etc. I tried to remove inconsistencies. I updated the last paragraph on matplotlib and saving files, since there was some redundant text, missing code etc.

**Comment by brownsarahm** [comment]
thanks @katrintirok I will merge these with my own (not pushed :/) changes that I made before teaching in March.

**Comment by annajiat** [comment]
Already merged with #159 
Thank you @brownsarahm @sechilds @katrintirok

**Pull request closed without merging**